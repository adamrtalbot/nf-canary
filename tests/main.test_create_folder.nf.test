nextflow_process {

    name "Test Process TEST_CREATE_FOLDER"
    script "main.nf"
    process "TEST_CREATE_FOLDER"

    test("Should run without failures") {

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
