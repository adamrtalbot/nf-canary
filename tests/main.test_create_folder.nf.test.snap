{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    [
                        "test1.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                        "test2.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
                    ]
                ],
                "outfolder": [
                    [
                        "test1.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                        "test2.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
                    ]
                ]
            }
        ],
        "timestamp": "2023-08-04T15:17:57+0000"
    }
}